integration_tests/redis/redisHelpers.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-await-in-loop */
+import { createRedisClient } from '../../server/data/redisClient'
+
+const rateLimitKeyPattern = 'rateLimit:*'
+
+const clearRateLimits = async () => {
```

Comment:
Just wipes any rate limit data in Redis: used to get a clean state before each test